### PR TITLE
Cue: Update Text, Stat, and Annolist panel cue schemas

### PR DESCRIPTION
**Pull request by joshhunt**
**What this PR does / why we need it**:

Updates the Text, Stat, and Annolist panel plugin cue schemas, and generate + uses the types in favor of manually crafted types. 

**Which issue(s) this PR fixes**:

Part of #48761

**Special notes for your reviewer**:

### CODE REVIEW

**Review by scottlepp — APPROVED** [review]

File: public/app/plugins/panel/annolist/module.tsx [hunk 1]
```diff
@@ -1,20 +1,19 @@
...
-export const plugin = new PanelPlugin<AnnoOptions>(AnnoListPanel)
+export const plugin = new PanelPlugin<PanelOptions>(AnnoListPanel)
   .setPanelOptions((builder) => {
     builder
       .addRadio({
         category: ['Annotation query'],
         path: 'onlyFromThisDashboard',
         name: 'Query filter',
-        defaultValue: false,
+        defaultValue: defaultPanelOptions.onlyFromThisDashboard,
```
Comment (JoaoSilvaGrafana) [inline public/app/plugins/panel/annolist/module.tsx:16]: Need to add these `defaultPanelOptions` to the other parts of the builder

File: public/app/plugins/panel/stat/module.tsx [hunk 2]
```diff
@@ -81,3 +85,101 @@ export const plugin = new PanelPlugin<StatPanelOptions>(StatPanel)
   .setPanelChangeHandler(statPanelChangedHandler)
   .setSuggestionsSupplier(new StatSuggestionsSupplier())
   .setMigrationHandler(sharedSingleStatMigrationHandler);
+
+function addStandardDataReduceOptions<T extends SingleStatBaseOptions>(
```
Comment (JoaoSilvaGrafana) [inline public/app/plugins/panel/stat/module.tsx:89]: This must have been what was committed by accident on the mob?

File: public/sass/theme.dark.generated.json [hunk 3]
```diff
@@ -152,6 +152,10 @@
     "menuTabs": {
       "height": 41
     },
+    "textHighlight": {
```
Comment (JoaoSilvaGrafana) [inline public/sass/theme.dark.generated.json:155]: Also this and `theme.light.generated.json` were committed accidentally

File: packages/grafana-schema/src/scuemata/dashboard/dist/family.cue [hunk 4]
```diff
@@ -27,6 +28,7 @@ import (
 Family: dashboard.Family & {
     compose: Panel: {
         // TODO do this with a loop once we include the panel type/plugin id in the model
+        annolist: pannolist.Panel
```
Comment (sdboyer) [inline packages/grafana-schema/src/scuemata/dashboard/dist/family.cue:31]: FYI - this is where we compose panel plugin schemas into the main dashboard schemas. This is all the old scuemata system, and will be removed and replaced by something similar when i finish grafana/thema#8

(tests caught this one, i'd forgotten about it myself while we were mobbing)

**Review by sdboyer — APPROVED** [review]

File: public/app/plugins/panel/stat/StatMigrations.ts [hunk 5]
```diff
@@ -1,17 +1,17 @@
 import { FieldColorModeId, FieldConfigSource, PanelModel } from '@grafana/data';
-import { sharedSingleStatPanelChangedHandler, BigValueGraphMode, BigValueColorMode } from '@grafana/ui';
-import { BigValueTextMode } from '@grafana/ui/src/components/BigValue/BigValue';
+import { BigValueTextMode , BigValueGraphMode, BigValueColorMode} from '@grafana/schema';
+import { sharedSingleStatPanelChangedHandler,} from '@grafana/ui';
```
Comment (mdvictor) [inline public/app/plugins/panel/stat/StatMigrations.ts:3]: Drone build *probably* fails here because of code styling.

**Review by mdvictor — APPROVED** [review]

**Pull request merged**